crates/bevy_pbr/src/material.rs

```diff
@@ -312,6 +321,33 @@ impl<M: Material, const I: usize> EntityRenderCommand for SetMaterialBindGroup<M
     }
 }
 
+pub fn verify_material_mesh_attributes<M: Material>(
```

Comment:
Instead of adding a new system, you could probably just call `required_vertex_attributes()` in `queue_material_meshes()`. 

Also, I personally would just panic instead of just an error. We already know this is an invalid state so I don't think we should keep running. Technically, the game would still run with the wrong attributes, but I'd rather just panic. That might be more controversial though. If you keep the error without panic, I would try to add a way to make sure we don't spam the console more than once with this.

Reply:
I feel like not panicking in this situation is more indicative of how bevy handles things like this. If attributes, textures, shaders, etc... are missing it currently doesn't panic.

Reply:
Well, I would also prefer if we panicked in all those cases too, but I don't want to push too hard for that. I don't really mind either way, as long as errors aren't just spammed every frame.

Reply:
Panicking in these cases seems counter to being able to hot-reload things like shaders, models, scenes, etc... Agree on not being spammed every frame.

Reply:
Having this as a separate system avoids spamming the error message -- the system only runs when the mesh or material is changed. (There's another error message about missing vertex attributes that _does_ get spammed, but that one is emitted when generating tangents for a mesh.)